app/controllers/todos_controller.rb

```diff
@@ -1,70 +1,50 @@
 class TodosController < ApplicationController
-  before_action :set_todo, only: %i[ show edit update destroy ]
+  before_action :set_todo, only: [:show, :edit, :update, :destroy]
 
-  # GET /todos or /todos.json
   def index
     @todos = Todo.all
```

Comment:
Consider adding pagination to the index action to improve performance when the number of todos grows large.